src/components/Search.tsx

```diff
@@ -34,6 +36,9 @@ type SearchProps = {
 };
 
 const sortableSearchTabs: SearchQuery[] = [CONST.TAB_SEARCH.ALL];
+const transactionItemMobileHeight = 100;
+const reportItemTransactionHeight = 52;
+const listItemPadding = 12; // this is equivalent to 'mb3' on every transaction/report list item
```

Comment:
NAB This feels brittle since if the height changes we would introduce the bug again. That being said, I'm not sure how to solve it this.

Reply:
Exactly same for me. We could in some way extract the same variable from styles perhaps and use, but still you would have to change it in two places.

I agree that its brittle but I see no other simple solution. padding affects item height, and correct item height has to be pushed down to RN List